src/profile-logic/process-profile.js

```diff
@@ -572,18 +588,15 @@ function _processMarkers(geckoMarkers: GeckoMarkerStruct): MarkersTable {
                 throw new Error('Unknown GCMajor status');
             }
           }
+          case 'Styles': {
```

Comment:
Question: is it a brand new type?

I think it would be useful to have a comment explaining that this type exists in profiles from newer gecko only, while profiles from older gecko will be of type "tracing".